delta/plugins/archive/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/archive/model/ArchiveRejection.scala

```diff
@@ -131,6 +135,22 @@ object ArchiveRejection {
     */
   final case class WrappedFileRejection(rejection: FileRejection) extends ArchiveRejection(rejection.reason)
 
+  /**
+    * Rejection returned when attempting to evaluate a command but the evaluation took more than the configured maximum value
+    */
+  final case class ArchiveEvaluationTimeout(command: CreateArchive, timeoutAfter: FiniteDuration)
+      extends ArchiveRejection(
+        s"Timeout while evaluating the command '${ClassUtils.simpleName(command)}' for archive '${command.id}' after '$timeoutAfter'"
+      )
+
+  /**
+    * Rejection returned when attempting to evaluate a command but the evaluation took more than the configured maximum value
+    */
+  final case class ArchiveEvaluationFailure(command: CreateArchive)
```

Comment:
We lose the error message from the underlying failure

Reply:
We don't loose it. Remember that any of those errors get logged already inside `EventSourceProcessor`. Here as a reason I just display what I want to expose to the client, which is not everything.